pkgs/games/prismlauncher/default.nix

```diff
@@ -60,7 +60,11 @@ stdenv.mkDerivation (finalAttrs: {
     "-DLauncher_BUILD_PLATFORM=nixpkgs"
   ] ++ lib.optionals (msaClientID != null) [ "-DLauncher_MSA_CLIENT_ID=${msaClientID}" ]
   ++ lib.optionals (lib.versionOlder qtbase.version "6") [ "-DLauncher_QT_VERSION_MAJOR=5" ]
-  ++ lib.optionals stdenv.isDarwin [ "-DINSTALL_BUNDLE=nodeps" "-DMACOSX_SPARKLE_UPDATE_FEED_URL=''" ];
+  ++ lib.optionals stdenv.isDarwin [
+    "-DINSTALL_BUNDLE=nodeps"
+    "-DMACOSX_SPARKLE_UPDATE_FEED_URL=''"
+    "-DCMAKE_INSTALL_PREFIX=${placeholder "out"}/Applications/"
```

Comment:
would this break the wrapper? or does wrapQtApps account for this?

otherwise lgtm :)

Reply:
Well, the wrapped derivation builds and runs fine on my aarch64-darwin

Reply:
but is PRISMLAUNCHER_JAVA_PATHS set? this is what allows the launcher to detect the java paths we include <https://github.com/NixOS/nixpkgs/blob/117f86f9fa6d01d1041ed9186ecbdcad9e42d57f/pkgs/games/prismlauncher/wrapper.nix#L119>

edit: it should be as `wrapQtAppsHook` does account for this
https://github.com/NixOS/nixpkgs/blob/921527b8658e1e319d40d6adf4f473343a738f64/pkgs/development/libraries/qt-6/hooks/wrap-qt-apps-hook.sh#L77